From a6e5f6825aabc4a98ef6d0d9f1accf076a41d2b5 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Mon, 6 Sep 2021 14:34:44 +0200
Subject: [PATCH] ARM: configs: bytedevkit: Fix boot logo

Link pwm_stm32 and stm32_timers statically.

Using them as modules means that they will be loaded too late and that
the boot logo can't be shown because pwm-backlight can't control the
pwm yet.
---
 arch/arm/configs/fragment-901-bytedevkit.config | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/fragment-901-bytedevkit.config b/arch/arm/configs/fragment-901-bytedevkit.config
index b45e5bf56e45..77734596ad19 100644
--- a/arch/arm/configs/fragment-901-bytedevkit.config
+++ b/arch/arm/configs/fragment-901-bytedevkit.config
@@ -9,3 +9,6 @@ CONFIG_LOGO_BAW_CLUT224=y
 CONFIG_DRM_FBDEV_EMULATION=y
 
 CONFIG_DP83867_PHY=y
+
+CONFIG_MFD_STM32_TIMERS=y
+CONFIG_PWM_STM32=y
-- 
2.20.1

